src/main/java/Parser.java

```diff
@@ -0,0 +1,154 @@
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Parser {
+
+    private String raw;
+    List<String> args;
+    String type;
+    String remainingArgs;
+    List<String> argsWithDate;
+
+    /**
+     * Constructor for Parser.
+     *
+     * @param move string that user types in
+     */
+    public Parser(String move) {
+        raw = move;
+        args = new LinkedList<>(Arrays.asList(move.split(" ")));
+        type = args.get(0);
+        args.remove(0);
+        remainingArgs = String.join(" ", args);
+    }
+
+    /**
+     * Gets for type of action.
```

Comment:
I like how you start your java doc comment with "Gets". Using simple present tense of verbs make the documentation easy to read.